File: supAndSub.c
#include <stdio.h>
#include <assert.h>

int sup(int num)
{
  return ++num;
}

int sub(int num){
  return --num;
}

int main()
{
  assert(5 == sub(6));
  assert(7 == sup(6));
  return 0;
}

File: JS/supAndSub.js
const assert = require('assert');

const sup = function(num) {
  return ++num;
}

const sub = function(num) {
  return --num;
}

try {
  assert.equal(5, sub(6));
  assert.equal(7, sup(6));
} catch (e) {
  console.log(e);
}